modules/local/pixelator/collect_metadata.nf

```diff
@@ -1,17 +1,12 @@
-import groovy.json.JsonOutput
-
-
 
 process PIXELATOR_COLLECT_METADATA {
     label 'process_single'
     cache false
 
-    conda "bioconda::pixelator=0.18.2"
+    conda "bioconda::pixelator=0.19.0"
```

Comment:
best to follow the standard practice of putting the package info in an `environment.yml` and provide that

Reply:
Done in https://github.com/nf-core/pixelator/pull/125